### PR TITLE
eightball.py

### PR DESCRIPTION
**Pull request by nuuls**
Kappa Keepo PogChamp xD 123

### CODE REVIEW

**Comment by dextermb** [comment]
Truly wonderful work! :fire:

**Comment by Nacht123** [comment]
**sub emotes** and **user bttv emotes** are in the list

File: pajbot/modules/eightball.py [hunk 1]
```diff
@@ -109,7 +115,7 @@ def eightball_command(self, **options):
         message = options['message']
 
         if message and len(message) > 0:
-            phrase = random.choice(self.phrases)
+            phrase = random.choice(self.phrases) + " " + random.choice(self.emotes)
```
Comment (pajlada) [inline pajbot/modules/eightball.py]: Using string concatenation like that is generally frowned upon.
You should instead keep phrase as it was before:
`phrase = random.choice(self.phrases)`
and then do
`emote = random.choice(self.emotes)`
and change the bot.me message to include {phrase} and then {emote} instead

**Comment by pajlada** [comment]
Looks good, thank you!

**Pull request merged**